### ISSUE TITLE
Logging stopped, web UI not accessible

### ISSUE THREAD

**Issue by ronnyandre** [adriankumpf/teslamate#920]
**Describe the bug**

Logging to Grafana stopped working. Checked the logs, that was referring to connection errors and shutdown. The frontend for Teslamate is not accessible, returning connection refused.

**Expected behavior**

It should run as normal, and keep logging. Web UI should also be accessible.

**Relevant entries from the logs**

```
08:34:31.841 [error] GenServer {Tortoise.Registry, {Tortoise.Connection, "TESLAMATE_302E3631363"}} terminating
** (stop) {:connection_failed, :not_authorized}
Last message: :connect
08:34:31.864 [info] Application teslamate exited: shutdown
{"Kernel pid terminated",application_controller,"{application_terminated,teslamate,shutdown}"}
Kernel pid terminated (application_controller) ({application_terminated,teslamate,shutdown})
Crash dump is being written to: erl_crash.dump...done
08:34:50.902 [info] Already up
```

**Environment**

- TeslaMate version: teslamate/teslamate:latest
- Type of installation: Docker
- OS TeslaMate is installed on: Raspbian
- User OS & Browser: MacOS 10.15.6
- Others:

**Comment by adriankumpf** [comment]
Looks like the the connection to the MQTT borker could not be established. Do you use your own broker? Are the credentials correct? You can try to disable MQTT with the `DISABLE_MQTT` environment variable: https://docs.teslamate.org/docs/configuration/environment_variables

**Comment by ronnyandre** [comment]
Might be the case actually. I am running Home Assistant on another server, with its internal MQTT broker. It failed on me last week and I reinstalled, and that have resetted the MQTT user settings. I was away during the weekend, and when I came back I tried configuring the MQTT settings, and suddenly TeslaMate worked again. I don't know if this had something with the MQTT settings, but it does correlate with what you are saying.
